packages/core/README.md

```diff
@@ -54,6 +54,20 @@ const client = await getSigningClient({
 });
 ```
 
+
+#### Cosm Wasm and Signing Cosm Wasm Client
+
+The `SeiSigningCosmWasmClient` and `SeiCosmWasmClient` can be used to interact with CosmWasm smart contracts. Typically, you need the `signingCosmWasmClient` to execute contract messages and the `cosmWasmClient` to query contract state.
+
+```typescript
+import { SeiCosmWasmClient, SeiSigningCosmWasmClient, connect } from '@sei-js/core';
```

Comment:
there's also helpers `getCosmWasmClient` and `getSigningCosmWasmClient`

Reply:
Where are these? I see those helper methods in the proto library, but that proto package didn't have the Tendermint 35 fixes I believe.

Reply:
Okay, cool I fixed this.